common/tekton/pipelines/build-images.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     default: tekton/images/httpd-ionic
   - name: OUTPUT_IMAGE_PROVIDER_CONFIGMAPKEY
     type: string
-    default: quay.io
+    default: IMAGE_PROVIDER
```

Comment:
@beekhof this seemed like a minor oversight from commit https://github.com/dagger-refuse-cool/blueprint-elements-library/commit/d26acb70a8aec7523d13d9ada997e91a3ad20bcd#diff-7ba4014d932474de034deb3c59516f623c0a2f3a90cb64160e30f9868f194651R17

It appears it should have been IMAGE_PROVIDER but was committed previously as just quay.io when all other common instances seem to reference IMAGE_PROVIDER. please check if this is a correct assumption and change back to the ConfigMap variable.

Reply:
I don't think this is correct, this would mean the default is the literal text "IMAGE_PROVIDER".

Reply:
Which could go one of two ways... would could consider "broken unless you override it" as a feature, but more likely we want to point to defaults that work

Reply:
Right below this, there is a similar variable with an IMAGE_ORG used as the default. If the case is true that you want to provide a default value, we'd have to change that as well

Reply:
I think we need to review this change a bit more to figure out the inconsistencies.

Reply:
Yep, my bad. I made the same mistake.

Reply:
I can confirm that we need values for these. So the default value has to be an actual value not a variable. e.g. default: quay.io not default: IMAGE_PROVIDER.  @ckupe if you can revert this change and chane IMAGE_ORG to the default value of daggerrefusecool that would be great.